plugins/action/action.cpp

```diff
@@ -113,4 +113,34 @@ float Action::get_max_speed_m_s() const
     return _impl->get_max_speed_m_s();
 }
 
+const char *Action::result_str(Result result)
```

Comment:
I thought I had moved that to `plugins/action/action_result.h`?

Reply:
Oops! Ok, I'll remove it.